src/Microsoft.CodeAnalysis.FxCopAnalyzers/Microsoft.CodeAnalysis.FxCopAnalyzers.sarif

```diff
@@ -2340,6 +2340,25 @@
             ]
           }
         },
+        "CA2012": {
+          "id": "CA2012",
+          "shortDescription": "Use ValueTasks correctly",
+          "fullDescription": "ValueTasks must only be consumed once, generally by directly awaiting them.  Attempts to consume a ValueTask multiple times may result in an exception or corruption.",
```

Comment:
Should the full description for CA2012 include anything about the AccessingIncompleteResultRule case? Maybe something like the following?

```suggestion
          "fullDescription": "ValueTasks must only be consumed once and should not have their result accessed until the instance has already completed. ValueTasks should generally be awaited. Attempts to consume a ValueTask multiple times may result in an exception or corruption.",
```

Reply:
Thanks.  Addressed.